static/css/style.css

```diff
@@ -127,28 +127,25 @@ img{
 	-moz-columns: 2;
 }
 
-/* to hide title in the second column of Members*/
-.transparent{
-	color: transparent;
-}
 
 /* paragraph*/
-.intro{
+.intro {
 	text-align: justify;
 	color: #5C5C5C;
-	font-size: 20px;
+	font-size: 12px;
```

Comment:
You have decreased the font size, but the "jumbotron p" from bootstrap os overwriting that.
You can this clearly see in Web inspector by inspecting you paragraph element.
Btw: do we really need jumbotron here?